src/services/game.service.ts

```diff
@@ -30,42 +29,58 @@ class GameService {
 
   async getRandomWord(id: string): Promise<string> {
     const game = await this.getById(id);
-    const vocabulary = await vocabularyService.getById(game.options.vocabularyId);
+    const vocabulary = await vocabularyService.getById(
+      game.options.vocabularyId,
```

Comment:
what is max length of line in your IDE? I see too many changes like this in your PRs, where one line is modified to 2-3-4 lines by this pattern